utils/lvm2/Makefile

```diff
@@ -40,8 +40,8 @@ define Package/lvm2
   CATEGORY:=Utilities
   SUBMENU:=Disc
   TITLE:=The Linux Logical Volume Manager
-  URL:=http://sourceware.org/lvm2/
-  DEPENDS:=+libdevmapper +libblkid +libreadline +libncurses
+  URL:=https://sourceware.org/lvm2/
+  DEPENDS:=+libdevmapper +libblkid +libreadline +libncurses +libaio
```

Comment:
Why the new dependency?

And wouldn't this require new `CONFIGURE_ARGS`?

Reply:
Build was failing on it.

There's a new version of LVM2. I will look into this soon.

Reply:
Asked and answered:

https://sourceware.org/git/?p=lvm2.git;a=blob;f=WHATS_NEW;h=0a0b6e925c0a967be4514f13abfca42f3d4b019e;hb=HEAD#l24

Reply:
Ah yes. I recall looking into the configure script to look for a way to get rid of it.